projects/storefrontlib/src/cms-components/cart/cart-details/cart-details.component.ts

```diff
@@ -61,9 +57,14 @@ export class CartDetailsComponent implements OnInit {
       )
     );
 
-    this.orderPromotions$ = this.promotionService.getOrderPromotions(
-      this.promotionLocation
-    );
+    this.orderPromotions$ = this.activeCartService
```

Comment:
maybe merge `cart.potentialOrderPromotions` and `cart.appliedOrderPromotions` from html.